doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-provider-con.adoc

```diff
@@ -0,0 +1,9 @@
+[id='business-central-security-management-provider-con_{context}']
+
+= Security management providers
+
+In the context of security management, a realm restricts access to different application resources. Realms contain information about the users, groups, roles, and permissions. A concrete user and group management service implementation for a specific realm is called a security management provider.
+
+If the built-in security management providers do not fit with your application security realm, then you can build and register your own security management provider.
```

Comment:
do not fit with your application security realm, change to:
do not meet the requirements of your application security realm

Reply:
Updated